script/jenkins/linux-test.sh

```diff
@@ -12,3 +12,6 @@ cleanup() {
 trap cleanup EXIT
 
 make update docker-system-tests
+
+# TODO(axw) make this part of the "system-tests" target
```

Comment:
I would do this when we remove the Python system tests, removing the `docker-system-tests` target and making `system-tests` target run the Go tests in the systemtest module.